dev_guide/builds.adoc

```diff
@@ -519,6 +519,7 @@ For example, defining a custom HTTP proxy to be used during build:
 ----
 ====
 
+// tag::using-a-proxy-for-git-cloning-1[]
```

Comment:
Probably should move this tag a few lines down to after "Using a Proxy for Git Cloning" title? The way it is now in the http_proxies topic, the rendered build is getting double headings back to back (both "Git Repository Access" and "Using a Proxy for Git Cloning").